tox.ini

```diff
@@ -54,11 +54,43 @@ commands =
     pip-audit {env:EXCLUDES:}
 
 [testenv:docs,docs-py{39,310,311,312}]
-description = Build documentation
+description = Build documentation by building the examples, checking the links and
+    performing quickbuild
 extras = docs
 passenv = BAYBE_DOCS_LINKCHECK_IGNORE
 setenv =
     SMOKE_TEST = true
 commands = 
+    # TODO Can we directly call the corresponding tox commands here?
```

Comment:
not sure that makes sense as they will be executed in a potentially different env

Reply:
Makes sense. I wanted to avoid having boilerplate code, hence my idea was to re-use the existing tox commands, but I agree that this is not what we actually want.

Reply:
then the TODO (which is a question anyway) should be removed